engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicDeploymentManager.scala

```diff
@@ -185,6 +185,11 @@ class PeriodicDeploymentManager private[periodic] (
     service.getStatusDetails(name).map(_.map(List(_)))
   }
 
+  override def getDeploymentStatusesToUpdate: Future[Map[newdeployment.DeploymentId, DeploymentStatus]] = {
+    // We don't handle this flow for k8s dm, so we return empty list map of updates
+    Future.successful(Map.empty)
```

Comment:
Yeah, I don't support synchronisation in this case as well but because of other reasons. I added more verbose explanation of why these DM's don't support synchronisation.